opentelemetry-api/src/opentelemetry/context/propagation/tracecontexthttptextformat.py

```diff
@@ -16,9 +16,12 @@
 import typing
```

Comment:
should tracecontext be moved to trace/propagation? It's a valid hierarchy. I don't see anything in the spec that calls out specifically where this should be: https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/library-layout.md

Reply:
makes more sense in `trace/propagation` than where it currently is. moved it.